DQM/SiPixelPhase1TrackClusters/src/SiPixelPhase1TrackClusters.cc

```diff
@@ -142,6 +157,7 @@ void SiPixelPhase1TrackClusters::analyze(const edm::Event& iEvent, const edm::Ev
         histo[ONTRACK_SIZE      ].fill(double(cluster.size()  ), id, &iEvent);
         histo[ONTRACK_POSITION_B].fill(clustgp.z(),   clustgp.phi(),   id, &iEvent);
         histo[ONTRACK_POSITION_F].fill(clustgp.x(),   clustgp.y(),     id, &iEvent);
+	histo[ONTRACK_SIZE_VS_ETA].fill(clustgp.eta(), cluster.sizeY(), id, &iEvent);
```

Comment:
This piece of code also shows that although not all the HistogramManagers are going to be used, because of the test for `is_ontrack`, they all get created. How much memory is that taking up? Seems like this would be better as a `std::unordered_map` where only the items from the configuration are created.